src/main/java/step3/RacingGame.java

```diff
@@ -0,0 +1,16 @@
+package step3;
+
+import step3.domain.Cars;
+import step3.view.RacingGameView;
+
+public class RacingGame {
+
+  public void start(int numberOfCar, int gameCount) {
+    Cars carList = new Cars(numberOfCar);
```

Comment:
```suggestion
    Cars cars = new Cars(numberOfCar);
```
Cars 타입에 이미 List의 뜻을 내포하고 있으니 변수명에 List는 빼도 되겠네요